client/settings/fraud-protection/advanced-settings/index.js

```diff
@@ -101,8 +107,20 @@ const FraudProtectionAdvancedSettingsPage = () => {
 	};
 
 	const handleSaveSettings = async () => {
-		if ( validateSettings( settings.advanced_fraud_protection_settings ) ) {
+		if ( validateSettings( advancedFraudProtectionSettings ) ) {
 			setIsSavingSettings( true );
+			if ( 'advanced' !== currentProtectionLevel ) {
```

Comment:
What do you think about moving the `advanced` to a constant?

Reply:
Moved in [773170d](https://github.com/Automattic/woocommerce-payments/pull/5685/commits/773170da25e7fd0251de70ebc538c9f68c15d8bf).